closed/custom/modules/java.base/Copy.gmk

```diff
@@ -177,6 +177,14 @@ $(call openj9_copy_files,, \
 	$(TOPDIR)/openj9-openjdk-notices \
 	$(LEGAL_DST_DIR)/openj9-openjdk-notices)
 
+ifeq (true,$(OPENJ9_ENABLE_JFR))
+
+$(call openj9_copy_files,, \
+	$(OPENJ9_TOPDIR)/runtime/metadata.blob \
+	$(LIB_DST_DIR)/metadata.blob)
+
+endif # OPENJ9_ENABLE_JFR
```

Comment:
Please move this after the DDR block (DDR is alphabetically before JFR).

Reply:
Moved it after `DDR` block.